src/Sylius/Behat/Context/Transform/ProductContext.php

```diff
@@ -35,6 +35,8 @@ public function __construct(ProductRepositoryInterface $productRepository)
     /**
      * @Transform /^product "([^"]+)"$/
      * @Transform /^"([^"]+)" product$/
+     * @Transform /^"([^"]+)" products$/
+     * @Transform /^(?:a|an) "([^"]+)"$/
      * @Transform :product
```

Comment:
If we would add `@Transform :secondProduct` we were be able to simplify a steps from my previous [comment](https://github.com/Sylius/Sylius/pull/4898/files#r62832772)